test/box/gh-6092-invalid-listen-uri.result

```diff
@@ -0,0 +1,128 @@
+env = require('test_run')
+---
+...
+test_run = env.new()
```

Comment:
You never use `env`. Please, do `test_run = require('test_run').new()`

Reply:
Fixed